systers_portal/community/views.py

```diff
@@ -21,6 +23,8 @@
 from community.models import Community, CommunityPage, RequestCommunity
 from users.models import SystersUser
 
+GOOGLE_MAPS_API_KEY = os.environ.get('GOOGLE_MAPS_API_KEY')
```

Comment:
could you load this through settings, as that keeps config separate from code.

Reply:
Sure I'll add this to the settings file

Reply:
Done